examples/internal/proto/globalconfig/globalconfig.buf.gen.yaml

```diff
@@ -0,0 +1,11 @@
+version: v1
+plugins:
+  - name: grpc-gateway
+    out: .
+    opt:
+      - paths=source_relative
+      - grpc_api_configuration=examples/internal/proto/globalconfig/grpc_annotations.yaml
```

Comment:
I don't think this option does anything to this generator?

Reply:
Which option? `-paths=source-relative`?

Reply:
Huh, nevermind, it's here: https://github.com/grpc-ecosystem/grpc-gateway/blob/9b09466d8afe9003306397242ba3d1e3fbcd4cae/protoc-gen-grpc-gateway/main.go#L29. I don't know why I left this comment.